exopy/measurement/hooks/addtask_hook.py

```diff
@@ -20,16 +20,23 @@ class AddTasksHook(BasePostExecutionHook):
     """Post-execusion hook to add a hierarchy of tasks.
 
     """
+    #: Reference to the root task at the base of the hierarchy
     root_task = Typed(RootTask)
+
+    #: Reference to the measurement workbench
```

Comment:
application workbench

The workbench is shared by the whole application and is not specific to the measurement plugin.